cyberbrain/value_stack.py

```diff
@@ -40,10 +40,20 @@ class ValueStackException(Exception):
 
 
 # Sometimes we need to put a _placeholder on TOS because we don't care about its value,
```

Comment:
Not sure if this is feasible, but my gut feeling is that these place holders can (and should) be unified as instances of `StackItem`. Could you try and see if it works?